proxy/proxy/httpd-conf/spacewalk-proxy.conf

```diff
@@ -20,6 +20,22 @@
     </IfVersion>
 </Directory>
 
+<Directory "/srv/www/htdocs/docs/*">
+    Options Indexes FollowSymLinks
```

Comment:
Try out what happens if you manually enter in the webbrowser "/docs/en/release-notes/" as path.
With "Options Indexes" you can see the directory index. Do we want this?
The option "FollwoSymlinks should only be set if we really have to deal with symlinks. Do we in this location?

Maybe just remove the full line?

Reply:
This works, we see the release notes file:
![image](https://user-images.githubusercontent.com/12104291/157866736-092109bf-8311-4666-a226-92a385325866.png)

I was not sure about this part, so I am glad you commented. I can remove the whole line. I think the release notes are always at the same location and are always named the same, so removing the options line would be a good idea.